docker-compose.yml

```
@@ -0,0 +1,43 @@
+services:
+  pillarbox-monitoring-dispatch:
+    build: ./pillarbox-monitoring-dispatch
+    image: srgssr/pillarbox-monitoring-dispatch:1.0.0
+    container_name: pillarbox-monitoring-dispatch
+    ports:
+      - "8080:8080"
+
+  pillarbox-monitoring-transfer:
+    build: ./pillarbox-monitoring-transfer
+    image: srgssr/pillarbox-monitoring-transfer:1.0.0
+    container_name: pillarbox-monitoring-transfer
+    ports:
+      - "8081:8081"
+    environment:
+      - PILLARBOX_QOS_OPENSEARCH_URI=http://opensearch:9200
+      - PILLARBOX_QOS_SSE_URI=http://pillarbox-monitoring-dispatch:8080/events
```

Following @defagos's suggestions:
```suggestion
      - PILLARBOX_MONITORING_OPENSEARCH_URI=http://opensearch:9200
      - PILLARBOX_MONITORING_SSE_URI=http://pillarbox-monitoring-dispatch:8080/events
```

---

Same as https://github.com/SRGSSR/pillarbox-monitoring-infra/pull/1#discussion_r1794921805